starboard/android/shared/android_info.cc

```diff
@@ -26,6 +30,17 @@ namespace shared {
 
 namespace {
 
+bool GetBuildFingerprint(char* out_value, int value_length) {
+  JniEnvExt* env = JniEnvExt::Get();
+  ScopedLocalJavaRef<jstring> id_string(env->CallStarboardObjectMethodOrAbort(
+      "getBuildFingerprint", "()Ljava/lang/String;"));
+  std::string utf_str = env->GetStringStandardUTFOrAbort(id_string.Get());
+  if (strlen(utf_str.c_str()) + 1 > value_length)
```

Comment:
should we truncate the value and copy value_length?

Reply:
This is fine, if the string is longer than 1k, it should be garbage.

Reply:
The Android SbSystemGetProperty() function leaves the buffer unmodified when the buffer is too small -- other platforms do this as well.